tour.php

```diff
@@ -14,7 +14,9 @@
 
 function tour_enqueue_scripts() {
 	static $once = false;
-	if ( $once ) {
+	$tours = apply_filters( 'tour_list', array() );
+
+	if ( $once || empty( $tours ) ) {
```

Comment:
Can we make this two ifs and set `$once = true` in between? because that way, if we happen to call this function more than once, each time all tours will be queried.